tests/playwright/sanity/editor-v2.test.js

```diff
@@ -0,0 +1,42 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../pages/wp-admin-page.js' );
+
+test.describe.only( 'Editor v2', () => {
```

Comment:
```suggestion
test.describe( 'Editor v2', () => {
```

we need to block it in the config

Reply:
😮 Done